panel/backends/wayland/kwin_wayland/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(NAME kwin_wayland_backend)
```

Comment:
As I mentioned in #2075, It would be better if drop `_backend` here, and tack-on `wmbackend_` on line 29/30..